spec/index.bs

```diff
@@ -8075,16 +8078,39 @@ dictionary GPUFragmentState
                     : {{GPUTextureSampleType/"uint"}}
                     :: |output| must have an unsigned integer scalar type.
                 </dl>
-            - If |colorState|.{{GPUColorTargetState/blend}} is [=map/exist|provided=] and
-                |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/color}}.{{GPUBlendComponent/srcFactor}}
-                or .{{GPUBlendComponent/dstFactor}} uses the source alpha
-                (is any of {{GPUBlendFactor/"src-alpha"}}, {{GPUBlendFactor/"one-minus-src-alpha"}},
-                or {{GPUBlendFactor/"src-alpha-saturated"}}), then:
-                - |output| must have an alpha channel (that is, it must be a vec4).
+            - If |colorState|.{{GPUColorTargetState/blend}} is [=map/exist|provided=]:
+                - If |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/color}}.{{GPUBlendComponent/srcFactor}} or
+                    |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/color}}.{{GPUBlendComponent/dstFactor}} or
+                    |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/alpha}}.{{GPUBlendComponent/srcFactor}} or
+                    |colorState|.{{GPUColorTargetState/blend}}.{{GPUBlendState/alpha}}.{{GPUBlendComponent/dstFactor}}
+                    uses the second input of the corresponding blending unit (is any of
+                    {{GPUBlendFactor/"src1"}}, {{GPUBlendFactor/"one-minus-src1"}},
+                    {{GPUBlendFactor/"src1-alpha"}}, {{GPUBlendFactor/"one-minus-src1-alpha"}}), then:
+                        - Set |UseDualSourceBlending| to `true`.
```

Comment:
This should be outside the step at line 8064 since it doesn't use `|output|`. I've moved it.